src/ui/components/button/Button.module.css

```diff
@@ -0,0 +1,16 @@
+.button {
```

Comment:
Have you tested how bundling works in dev and production mode? Does it all get bundled into a single css file in production mode?